README.md

```diff
@@ -82,23 +81,18 @@ You can use a `.solhint.json` file to configure Solhint for the whole project.
 To generate a new  sample `.solhint.json` file in current folder you can do:
 
 ```sh
-solhint init-config
+solhint --init 
 ```
 
 This file has the following
 format:
 
-
 ```json
   {
-    "extends": "solhint:recommended",
-    "plugins": [],
-    "rules": {
-      "avoid-suicide": "error",
-      "avoid-sha3": "warn"
-    }
+    "extends": "solhint:default"
```

Comment:
so, this change can be rolled back?

Reply:
Yes no problem, commit c840abd rolls this change back.